### ISSUE TITLE
hosts page - incorrect layout after photo change

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#377]
Alex changed his profile photo in #373 
This seems to have caused layout issues 

![image](https://user-images.githubusercontent.com/7018928/188037073-02b3a664-89a6-45aa-9ec1-2d5c9a4ab3a1.png)

@ironicbadger can you tell us where you got the photo? I presume you copied it from some other online profile where the image has been shrunk.

My theory:
I presume given it's data size - 29 KB - and it's pixel dimensions - 300x263 - that a too-small image size applied to a profile causes layout problems instead of being scaled/interpolated (despite pixelation).

@ironicbadger can you procure a higher-resolution copy relatively easily? Then again, this may not be the solution, but is a bandaid...

**Comment by elreydetoda** [comment]
@ChanceM (guessing that's them, since same name in matrix) posted this:

>  Looking at issue #377, which led down a deeper rabbit hole. Do you know why hosts/guests are not rendered with the people layout? Right now they are rendered with the default list layout, not sure if they was by design.

my brief response (more later):

> tl;dr: IIRC since there is no content in the content/hosts/ (& guests) then there is no way to apply a list page to it besides through the default list page (at least from what I could tell)
>
>So, that's why I [check for hosts/guests](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/24a2507059aea540778dbef0c740ffac4be0aba9/themes/jb/layouts/_default/list.html#L14)
>
>Plus the people's page lists both out, so it's got an [extra range](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/24a2507059aea540778dbef0c740ffac4be0aba9/themes/jb/layouts/people/list.html#L14) for the groups of people.

**Comment by ChanceM** [comment]
> @ChanceM (guessing that's them, since same name in matrix) posted this:
> 
> >  Looking at issue #377, which led down a deeper rabbit hole. Do you know why hosts/guests are not rendered with the people layout? Right now they are rendered with the default list layout, not sure if they was by design.
> 
> my brief response (more later):
> 
> > tl;dr: IIRC since there is no content in the content/hosts/ (& guests) then there is no way to apply a list page to it besides through the default list page (at least from what I could tell)
> >
> >So, that's why I [check for hosts/guests](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/24a2507059aea540778dbef0c740ffac4be0aba9/themes/jb/layouts/_default/list.html#L14)
> >
> >Plus the people's page lists both out, so it's got an [extra range](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/24a2507059aea540778dbef0c740ffac4be0aba9/themes/jb/layouts/people/list.html#L14) for the groups of people.

Okay changing the type from host to people in the content/hosts/_index.md will make it render with the people template and adding the same for guests will do the same. 

For the duplicate of a host as a guest we would need to duplicate the person but set the type to guest.

**Comment by ChanceM** [comment]
@elreydetoda, @gerbrent  I have a working version in a [branch](https://github.com/ChanceM/jupiterbroadcasting.com/tree/bug/fix-hosts-layout). It would require some rework for those that are hosts and guests but it would limit the need to retype contact info twice.

**Comment by elreydetoda** [comment]
Hey @ChanceM , I wasn't positive about this earlier but I was just able to test it out locally. It seems like you're trying to solve #376  and this issue with your branch.

While I'm all for trying to get the most results with a PR, I think in this instance we might want to split them up. #376 (just off the top of my head) will require at minimum [scraper](https://github.com/jupiterBroadcasting/show-scraper/) changes. While they might seem insignificant the scraper isn't the most modular so it'll take some time.


For this issue though, you already have everything you need to close it. It's just the styling changes you made to/for the cards already. I was able to [cherry-pick](https://stackoverflow.com/a/32121718) 2 of your commits and then replicated the only other styling change you'd need on this branch: https://github.com/elreydetoda/jupiterbroadcasting.com/tree/just_astetics

If you could just do a PR with only the changes you made for aesthetics (i.e. what I picked/replicated in my branch), then we could definitely close this issue with that PR.

![image](https://user-images.githubusercontent.com/10230166/188789203-321c40f2-6b0d-4b34-a75c-5e9473132ab0.png)

Thank you for taking the time to figure this out :partying_face: , and I'll comment on #376 about some of the questions I have about your approach for solving that issue (i.e. the rest of what you were suggesting with the branch you'd mentioned)

**Comment by ChanceM** [comment]
Well part of those changes incorporating fixing it to use the correct template. Should that portion be incorporated in #376 or into a new issue?

**Comment by elreydetoda** [comment]
yes, that should be in #376 

closed with #397

**Comment by gerbrent** [comment]
For future reference:

most of the images were scraped by the scraper from Fireside. The images thus were necessarily previously ingested by Fireside's process and likely made to conform to a specific standard.

Alex' photo is the first to be replaced independent of Fireside - he applied it here as a PR - and thus is the first case of integrating non-standard photos into our site here.

THAT is the difference, and why all a sudden Alex' photo is problematic.

Likely we will need to test a variety of images to see how Hugo & company manage their integration seamlessly.

_Originally posted by @gerbrent in https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/409#discussion_r966442608_
